CHANGELOG.md

```diff
@@ -120,7 +122,8 @@ property: <top> <right> <bottom> <left>
 * Initial release
 
 
-[Unreleased]: https://github.com/Shopify/stylelint-config-shopify/compare/v3.0.1...HEAD
+[Unreleased]: https://github.com/Shopify/stylelint-config-shopify/compare/v3.0.2...HEAD
+[3.0.2]: https://github.com/Shopify/stylelint-config-shopify/compare/v2.1.0...v3.0.2
```

Comment:
This should be `v3.0.1...v3.0.2`

I just did it on master - sorry for the mixup (thought it was something you forgot, since v3.0.2 has already been tagged).